dev_guide/cron_jobs.adoc

```diff
@@ -127,14 +129,30 @@ Doing this prevents them from generating unnecessary artifacts.
 [[cleaning-up-after-a-cron-job]]
 == Cleaning Up After a Cron Job
 
-Cron jobs can leave behind artifact resources such as jobs or pods. Check if any remain:
+Cron jobs can leave behind artifact resources such as jobs or pods.  As a user it is important
```

Comment:
For `Cron jobs can leave behind artifact resources such as jobs or pods.` would it be okay to change it to `Cron jobs can leave behind artifacts such as jobs or pods.`?